src/calculateRentalCost.js

```diff
@@ -4,7 +4,15 @@
  * @return {number}
  */
 function calculateRentalCost(days) {
-  // write code here
+  const pricePerDay = 40;
+
+  if (days < 3) {
+    return pricePerDay * days;
```

Comment:
Instead of using an `else if`, you can simplify the control flow by just using an `if` with a `return` statement since the code execution will not proceed past the `return`.